recipes-tpm2-tss/tpm2-tss/tpm2-tss_%.bbappend

```diff
@@ -0,0 +1,2 @@
+FILES_${PN} = "/lib/udev"
```

Comment:
@Asiderr What is the status of this upstream patch?

Reply:
@macpijan Sent and stuck.